indico/modules/events/papers/blueprint.py

```diff
@@ -32,6 +32,8 @@
 _bp.add_url_rule('/manage/papers/', 'management', management.RHPapersDashboard)
 _bp.add_url_rule('/manage/papers/teams', 'manage_teams', management.RHManagePaperTeams,
                  methods=('GET', 'POST'))
+_bp.add_url_rule('mamange/papers/enable/<reviewing_type>', 'switch', management.RHSwitchReviewingType,
```

Comment:
- `mamange`? 🐒 
- `.../<any(content,layout):reviewing_type>` - then you don't need to check against a list of valid types in the controller anymore